codersdk/workspacebuilds.go

```diff
@@ -73,6 +73,7 @@ type WorkspaceBuild struct {
 	Status                  WorkspaceStatus      `json:"status" enums:"pending,starting,running,stopping,stopped,failed,canceling,canceled,deleting,deleted"`
 	DailyCost               int32                `json:"daily_cost"`
 	MatchedProvisioners     *MatchedProvisioners `json:"matched_provisioners,omitempty"`
+	TemplateVersionPresetID *uuid.UUID           `json:"template_version_preset_id" format:"uuid"`
```

Comment:
I would mark this field as optional, as it won't be a standard case for a workspace build to use a preset. Thoughts?

Reply:
done

Reply:
Should we add `omitempty` as well? `MatchedProvisioners` is also nullable and has it.